src/components/pages/devices/devices.container.js

```diff
@@ -1,6 +1,25 @@
 // Copyright (c) Microsoft. All rights reserved.
 
+import { connect } from 'react-redux';
 import { translate } from 'react-i18next';
 import { Devices } from './devices';
+import {
+    epics as iotHubManagerEpics,
+    getDevices,
+    getDevicesError
+} from 'store/reducers/iotHubManagerReducer';
 
-export const DevicesContainer = translate()(Devices);
+
+// Pass the devices status
+const mapStateToProps = state => ({
+    devices: getDevices(state),
+    error: getDevicesError(state)
+});
+
+// Wrap the dispatch method
+const mapDispatchToProps = dispatch => ({
+    refresh: () => dispatch(iotHubManagerEpics.actions.fetchDevices()),
```

Comment:
These functions are the same only one should be passed. Maybe rename it to `fetchDevices`.

Reply:
Haha.  You added the second one.  But yes, you're right... we shouldn't pass two of the same thing.